app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,35 @@
+<div class="show-container">
```

Comment:
I see that we use it only to apply `text-align: center` styling. We can get rid of this `div` and instead use `text-center` CSS class in places where we really need it.